android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt

```diff
@@ -496,6 +497,48 @@ class PlayerModule(private val context: ReactApplicationContext) : ReactContextB
         }
     }
 
+    /**
+     * Initiates casting the current video to a cast-compatible remote device. The user has to choose to which device it
+     * should be sent.
+     */
+    @ReactMethod
+    fun castVideo(nativeId: NativeId) {
+        uiManager()?.addUIBlock {
+            players[nativeId]?.castVideo()
+        }
+    }
+
+    /**
+     * Stops casting the current video. Has no effect if [isCasting] is false.
+     */
+    @ReactMethod
+    fun castStop(nativeId: NativeId) {
+        uiManager()?.addUIBlock {
+            players[nativeId]?.castStop()
+        }
+    }
+
+    /**
+     * Whether casting to a cast-compatible remote device is available. [PlayerEvent.CastAvailable] signals when
+     * casting becomes available.
+     */
+    @ReactMethod
+    fun isCastAvailable(nativeId: NativeId, promise: Promise) {
+        uiManager()?.addUIBlock {
+            promise.resolve(players[nativeId]?.isCastAvailable)
```

Comment:
Should we fallback to `false` if no player instance could be found?

Reply:
Valid remark. I do think my current solution is in line with the existing patterns (e.g. `isAd` or `isLive`). In addition it does make sense semantically in my opinion. 

(In theory we could even return the correct value from the `BitmovinCastManager` even if we don't have a player but that's not in line with what we have on native.)

Reply:
Having a fallback is probably not ideal. Missing the player instance is a serious problem and may not happen as it is a broken fundamental framework.

Defining and implementing a proper solution is independent of the current changes.